src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -278,4 +279,5 @@
             value="${data.account.googleId}">
         <input type="hidden" name="<%= Const.ParamsNames.SESSION_TOKEN%>" value="${data.sessionToken}">
     </form>
+    <div name="<%= Const.ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT %>" data-value="${fsEnableEdit}"></div>
```

Comment:
Apologies I wasn't clear; the idea I had in mind was to embed the data in a data attribute of an existing element. The `form` element at the very top seems ideal for this purpose. The name of the data attribute can be `data-editsessiondetails`; the JavaScript can then do a `$("#form_feedbacksession").data("editsessiondetails")` (substitute with the relevant constants).